src/main/java/org/broadinstitute/hellbender/tools/walkers/coverage/DepthOfCoverage.java

```diff
@@ -90,6 +90,8 @@
     private Map<Locatable, DepthOfCoveragePartitionedDataStore> activeCoveragePartitioner = new HashMap<>();
```

Comment:
Add the following disclaimer about the gene summary file to an appropriate place in the docs: `in its current incarnation it will count genes not covered by the intervals as non-existent `